app/main.py

```diff
@@ -1,8 +1,24 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    persons = []
+
+    for person_data in people:
+        persons.append(Person(person_data["name"], person_data["age"]))
+
+    for person_data in people:
+        person = Person.people[person_data["name"]]
+
+        if "wife" in person_data and person_data["wife"]:
+            person.wife = Person.people[person_data["wife"]]
```

Comment:
Assigning `person.wife` assumes that the `wife` attribute exists on the `Person` object. You should initialize this attribute in the `Person` class to avoid potential `AttributeError`.